qiskit/ignis/verification/randomized_benchmarking/circuits.py

```diff
@@ -133,9 +144,11 @@ def randomized_benchmarking_seq(nseeds: int = 1,
                                 length_multiplier: Optional[List[int]] = 1,
                                 seed_offset: int = 0,
                                 align_cliffs: bool = False,
-                                interleaved_gates: Optional[List[List[str]]] = None,
```

Comment:
The current type of an `interleaved_elem` is:
`Optional[Union[List[qiskit.QuantumCircuit], List[qiskit.quantum_info.operators.symplectic.Clifford], List[CNOTDihedral]]]
`
So, the possible input could be given either as a list of `QuantumCircuit` or as a list of `Clifford ` or  `CNOTDihedral` objects.
The original type was `List[List[str]]`.
I'm not sure if it's worth the effort to write a special function that translates such a list (e.g. 
`[['x 0', 'x 1', 'cx 0 1'], ['x 0']]` into a list of `QuantumCircuit`)
Especially since the interleaved RB tutorial notebook does not exist now.
Anyway, I added a relevant Error if the type is wrong.

Reply:
This was basically a 'temporary API' until we will have a proper Clifford class.

Reply:
Yes, but we still need to maintain some deprecation, so there should still be a deprecated `interleaved_gates` arg that if not set to None raises the error. Something like

```
if interleaved_gates is not None:
    raise Deprecated("`interleaved_gates` kwarg is deprecated. Please use the `interleaved_elem` kwarg that supersedes it.")
```

Reply:
OK, done.